recipes/optuna/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "optuna" %}
+{% set version = "0.18.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7f7d92d8d1a3e0c5c3b92b4272a29b110a4a3d53e140a55e564c8756225a0c99"
+
+build:
+  number: 0
+  entry_points:
+    - optuna = optuna.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be `noarch: python`?

Reply:
Yes, I added `noarch: python` (https://github.com/conda-forge/staged-recipes/pull/10081/commits/e55bb0a2035901450272991537092ec3b6313b9f).
Thanks!